test/llvm_autogenerated/invoke_wrapper.s

```diff
@@ -0,0 +1,201 @@
+	.text
```

Comment:
I added a semi-handwritten test to dot_s (it was not handwritten from scratch, but I kind of removed all unnecessary parts from a simple generated file, resulting in a simple test case).